etcd-backup-operator/cmd/backup-operator/main.go

```diff
@@ -0,0 +1,83 @@
+package main
+
+import (
+	"os"
+	"runtime"
+	"time"
+
+	"github.com/coreos/etcd-operator/etcd-backup-operator/version"
+	"github.com/coreos/etcd-operator/pkg/util/k8sutil"
+
+	"github.com/Sirupsen/logrus"
+	"k8s.io/api/core/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/scheme"
+	v1core "k8s.io/client-go/kubernetes/typed/core/v1"
+	"k8s.io/client-go/tools/leaderelection"
+	"k8s.io/client-go/tools/leaderelection/resourcelock"
+	"k8s.io/client-go/tools/record"
+)
+
+func main() {
+	namespace := os.Getenv("MY_POD_NAMESPACE")
```

Comment:
Put all these into a func ?

Reply:
`namespace`, `name`,  and `id` are being used by code down below.

Reply:
ok